tests/documentation/command-reference/doc_command_reference_createnamespace_test.go

```diff
@@ -0,0 +1,64 @@
+package docautomation
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/google/go-cmp/cmp"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-developer/odo/tests/helper"
+)
+
+var _ = Describe("doc command reference odo create namespace", func() {
+	var commonVar helper.CommonVar
+	var commonPath = filepath.Join("command-reference", "docs-mdx", "create-namespace")
+	var outputStringFormat = "```console\n$ odo %s\n%s```\n"
+
+	BeforeEach(func() {
+		commonVar = helper.CommonBeforeEach()
+		helper.Chdir(commonVar.Context)
+		Expect(helper.VerifyFileExists(".odo/env/env.yaml")).To(BeFalse())
+	})
+
+	AfterEach(func() {
+		helper.CommonAfterEach(commonVar)
+	})
+
+	Context("To create a namespace resource", func() {
+
+		BeforeEach(func() {
```

Comment:
```suggestion
		AfterEach(func() {
```
I think adding an AfterEach is required because we must restore the state of the cluster by deleting the created namespace.
And it might be better to do that instead of BeforeEach.